moto/ec2/responses/elastic_block_store.py

```diff
@@ -32,10 +32,13 @@ def create_volume(self):
         size = self._get_param('Size')
         zone = self._get_param('AvailabilityZone')
         snapshot_id = self._get_param('SnapshotId')
+        tags = self._parse_tag_specification("TagSpecification")
+        volume_tags = tags.get('image', {})
```

Comment:
This isn't correct - it should be 'volume'. 

Of `ResourceType`, the docs say:
> The type of resource to tag. Currently, the resource types that support tagging on creation are instance and volume .

Calling `create_volume` using `image` causes an exception:
`botocore.exceptions.ClientError: An error occurred (InvalidParameterValue) when calling the CreateVolume operation: 'image' is not a valid taggable resource type for this operation.`

Reply:
I've opened #1448 to correct it.